### ISSUE TITLE
Awkwardly labeled Yes/No switch toggles in Home Feed Preferences

### ISSUE THREAD

**Issue by xnuk** [bluesky-social/social-app#2818]
### Describe the bug
![image](https://github.com/bluesky-social/social-app/assets/3071003/e96edf52-158e-4b81-a9e4-31a68478434d)

> Set this setting to "No" to hide all reposts from your feed.

But there's no "No" button. Plus switching label between "Yes" and "No" may confuse users. Generally I don't think it isn't great idea to put Y/N label literally in checkbox-like-UI.

### To Reproduce
Go to `Settings` :arrow_forward: `Home Feed Preferences` in the app.

### Expected behavior
An example could be like:
![image](https://github.com/bluesky-social/social-app/assets/3071003/e1068730-1912-43e6-894b-bd52b17c5581)

### Details
- `https://bsky.app/` `Build version: 1.68.0; Platform: web`
- on Firefox 123
- on Arch Linux x64

**Comment by mschwendener** [comment]
I agree that the toggles should look differently to make it clearer if they are active or deactivated, also in light mode.

https://www.nngroup.com/articles/toggle-switch-guidelines 

https://www.justinmind.com/ui-design/toggle-patterns-examples

**Comment by jamesleesaunders** [comment]
I assume this is no longer an issue as this screen has been redesigned?
https://bsky.app/settings/following-feed

And now looks like this:

Mobile:
![Image](https://github.com/user-attachments/assets/9aa441fd-78b3-4ce3-b774-14d423776e07)

Web:
<img width="606" alt="Image" src="https://github.com/user-attachments/assets/fda6bf46-e90b-4b80-8ac4-628b90bee9e3" />

Can this be closed?
